### PR TITLE
fix: fix web wallet smoketest

### PR DESCRIPTION
**Pull request by Extheoisah**

### CODE REVIEW

File: ci/tasks/web-wallet-smoketest.sh [hunk 1]
```diff
@@ -13,7 +13,8 @@ for i in {1..15}; do
   echo "Attempt ${i} to curl web wallet"
   curl --location -f ${host}:${port}
   status=$?
-  curl --location -f ${web_wallet_mobile_host}:${port}
+  if [[ web_wallet_mobile_host == "" ]]; then echo "No web wallet secondary host available";
```
Comment (bodymindarts) [inline ci/tasks/web-wallet-smoketest.sh:16]: This reads like its a negative even though it is expected.
```suggestion
  if [[ web_wallet_mobile_host == "" ]]; then echo "Skipping secondary host - not configured."; 
```

Comment (bodymindarts) [reply ci/tasks/web-wallet-smoketest.sh:16]: I think you've got the syntax wrong:
```
if [[ .. ]]; then
  true thing
 else
   false thing
 fi
```

Comment (Extheoisah) [reply ci/tasks/web-wallet-smoketest.sh:16]: @bodymindarts Like this right
```
if [[ web_wallet_mobile_host != "" ]]; then
    curl --location -f ${web_wallet_mobile_host}:${port};
  else
    echo "Skipping secondary host - not configured.";
  fi
  ```

Comment (bodymindarts) [reply ci/tasks/web-wallet-smoketest.sh:16]: You shouldn't need the `;` at the end of the lines. And formatting (indentation) is off.

**Pull request merged**